template_app_config_local.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+#
+# Script for templating the jinja2 app-config.local.yaml file.
+#
+# It collectes all the parameters passed with the - switch into jinja2 
+#  key/value variables that are to be replaced inside the jinja2 template.
+
+set -o errexit
+
+show_usage () {
+    echo ""
+    echo >&2 "$@"
+    echo ""
+    echo "Script for templating the jinja2 app-config.yaml file."
+    echo ""
+    echo "For required parameters check the jinja2 template file being used."
+    echo "The default template file is located at the ./manifest/app-config.local.yaml.j2" file.
+    echo ""
+    exit 1
+}
+
+JINJA2_TEMPLATE_VARIABLES=""
+INVALID_SWITCH=""
+
+set +e
+while [ $# -gt 0 ]; do
+    #echo "$1"
+    if [[ $1 == "--"* ]]; then
+        case $1 in
+            --help) show_usage; break 2 ;;
+            *) show_usage "Invalid switch $1" ; break 2 ;;
+        esac
+    elif [[ $1 == "-"* ]]; then
+        param="${1/-/}";
+        shift;
+        value="${1}";
+        echo "param: ${param}, value: ${value}"
+        shift;
+        JINJA2_TEMPLATE_VARIABLES+=" -D${param}=${value}"
```

Comment:
Can you document how we should use this script and pass the parameters ?

Reply:
Added some documentation to the README.md file. Could you check if it's understandable and enough?